pkg/api/vlabs/azenvtypes.go

```diff
@@ -35,6 +35,7 @@ type DCOSSpecConfig struct {
 //KubernetesSpecConfig is the kubernetes container images used.
 type KubernetesSpecConfig struct {
 	KubernetesImageBase              string `json:"kubernetesImageBase,omitempty"`
+	HyperkubeImageBase               string `json:"hyperkubeImageBase,omitempty"`
```

Comment:
should this also be added to https://github.com/Azure/aks-engine/blob/master/pkg/engine/types.go#L49 ?